### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Ishchuk-R**
[DEMO LINK](https://Ishchuk-R.github.io/layout_search-bar-airbnb/)
[TEST REPORT LINK](https://Ishchuk-R.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+.input-search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input-search:focus {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  font-weight: 500;
+}
+
+.input-lg {
```
Comment (earthhuman41) [inline src/style.css]: We don't have any basic search bar, so why we should use there big and small search class names? We have search-input and search-input - small (--small). Therefore, we set all styles that are supposed to be for the "large" input for all inputs in general, and with the modifier small we change only the necessary values to make our input smaller.

Comment (Ishchuk-R) [reply src/style.css]: fixed it

**Comment by earthhuman41** [comment]
Clear icon too close to right border, make gap for it
Screen Shot 2020-02-27 at 11 11 19 AM

**Comment by Ishchuk-R** [comment]
> Clear icon too close to right border, make gap for it
> Screen Shot 2020-02-27 at 11 11 19 AM

**Comment by Ishchuk-R** [comment]
done

**Pull request closed without merging**